src/platform/plugins/shared/chart_expressions/expression_partition_vis/public/utils/layers/get_color.test.ts

```diff
@@ -12,401 +12,397 @@ import { chartPluginMock } from '@kbn/charts-plugin/public/mocks';
 import { byDataColorPaletteMap, SimplifiedArrayNode } from './get_color';
 import type { SeriesLayer } from '@kbn/coloring';
 import { dataPluginMock } from '@kbn/data-plugin/public/mocks';
-import { fieldFormatsMock } from '@kbn/field-formats-plugin/common/mocks';
 import type { DataPublicPluginStart } from '@kbn/data-plugin/public';
 import { getColor } from './get_color';
 import { createMockVisData, createMockBucketColumns, createMockPieParams } from '../../mocks';
-import { generateFormatters } from '../formatters';
 import { ChartTypes } from '../../../common/types';
 import { getDistinctSeries } from '..';
+import { getColorCategories } from '@kbn/chart-expressions-common';
 
-describe('#byDataColorPaletteMap', () => {
-  let paletteDefinition: PaletteDefinition;
-  let palette: PaletteOutput;
-  const visData = createMockVisData();
-  const defaultFormatter = jest.fn((...args) => fieldFormatsMock.deserialize(...args));
-  const formatters = generateFormatters(visData, defaultFormatter);
+describe('get color', () => {
```

Comment:
It looks like a lot has changed here but most of it is just adding this top level `describe` I think all test files should have a top level to easily run everything.